### PR TITLE
Document service options in default.spec

### PR DESCRIPTION
**Pull request by Julian-O**
Also trivial copy-edits to other comments.

### CODE REVIEW

**Comment by Julian-O** [comment]
Slipped in another comment to explain how to find what bootstrap options were valid.

File: buildozer/default.spec [hunk 1]
```diff
@@ -58,7 +58,12 @@ requirements = python3,kivy
 # Valid options are: landscape, portrait, portrait-reverse or landscape-reverse
 orientation = portrait
 
-# (list) List of service to declare
+# (list) List of services to declare
+# Each service consists of a name (a valid Java class name, with the first letter capitalized)
+# followed by a colon, followed by the name of the Python script (.py file) that should be
+# launched. This is optionally followed by ":foreground" for foreground services or
+# ":foreground:sticky" for sticky foreground services. The default is a background service.
+# Bound services are not supported.
```
Comment (misl6) [inline buildozer/default.spec:66]: Maybe we want to say that (at least ATM) this is an android-specific feature, even if does not have the `android.*` prefix?

Comment (Julian-O) [reply buildozer/default.spec:66]: Ah, I hadn't even noticed it didn't have an android prefix. I am on it.

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**